ml-data-hub-plugin/src/test/groovy/com/marklogic/gradle/task/BaseTest.groovy

```diff
@@ -283,6 +315,103 @@ class BaseTest extends Specification {
         return databseFragment.getElementValues("//m:range-path-index").size()
     }
 
+    void deleteRangePathIndexes(String databaseName) {
+        String databaseFragment = getDatabaseManager().getPropertiesAsJson(databaseName)
+        ObjectMapper mapper = new ObjectMapper()
+        JsonNode dbObject = mapper.readTree(databaseFragment)
+        JsonNode rangePathIndexes = dbObject.get("range-path-index")
+        for(JsonNode rangePathIndex : rangePathIndexes) {
+            String type = rangePathIndex.get("scalar-type").asText()
+            String collation = rangePathIndex.get("collation").asText()
+            String path = rangePathIndex.get("path-expression").asText()
+            String pos = rangePathIndex.get("range-value-positions").asText()
+            String val = rangePathIndex.get("invalid-values").asText()
+            deleteRangePathIndexes(databaseName, type, collation, path, pos, val)
+        }
+    }
+
+    void deleteRangePathIndexes(String databaseName, String type, String collation, String pathSeq,
+            String pos, String val) {
+        ServerEvaluationCall eval = hubConfig().newStagingClient().newServerEval()
+
+        if(pos.equals("false")) {
+            pos = "fn:false()";
+        } else {
+            pos = "fn:true()";
+        }
+
+        String installer ='''
+                import module namespace admin = \"http://marklogic.com/xdmp/admin\" at \"/MarkLogic/admin.xqy\";
```

Comment:
this is kind of horrible looking, any alternate way to construct xquery?  it will be hard to maintain.

Reply:
will this run as hub admin even?  id be surprised if this works without more privileges, but id be happy to be proven wrong.

Reply:
I will reconstruct the query in a better way. Yes, this query works with hub-admin-user. I verified in the query-console (added qconsole-user role from admin ui).
hub-admin-role has admin-database privilege which can delete indexes from a given database.